src/blueapi/worker/reworker.py

```diff
@@ -15,6 +15,7 @@
     EventStream,
     WatchableStatus,
 )
+from blueapi.worker.worker_busy_error import WorkerBusyError
```

Comment:
I would suggest a relative import since `WorkerBusyError` is never actually leaving the `worker` package

```suggestion
from .worker_busy_error import WorkerBusyError
```